src/components/Scheme/SchemeContainer.tsx

```diff
@@ -63,9 +87,11 @@ export default class SchemeContainer extends React.Component<IProps & RouteCompo
           <div className={css.schemeMenu}>
             <Link className={proposalsTabClass} to={`/dao/${daoAvatarAddress}/scheme/${scheme.id}/proposals/`}>Proposals</Link>
             <Link className={infoTabClass} to={`/dao/${daoAvatarAddress}/scheme/${scheme.id}/info/`}>Info</Link>
-            <Link className={css.createProposal} to={`/dao/${daoAvatarAddress}/scheme/${scheme.id}/proposals/create`} data-test-id="createProposal">
-              + New proposal
-            </Link>
+            <a className={css.createProposal}
+              data-test-id="createProposal"
+              href="#"
```

Comment:
this will add a # to the URL, thats the issue you are seeing. better to do href="javascript:void(0)"

Reply:
fixed